webpack.config.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable */
+
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  mode: 'development',
+  entry: './src/index.js',
+  devServer: {
+    static: './dist',
+  },
+  plugins: [
+    new HtmlWebpackPlugin({
+      template: './src/index.html',
+    }),
+  ],
+  output: {
+    filename: 'main.js',
+    path: path.resolve(__dirname, 'dist'),
+  },
+  module: {
+    rules: [
+      {
+        test: /\.css$/i,
+        use: ['style-loader', 'css-loader'],
+      },
+    ],
+  },
+  // optimization: {
+  //  runtimeChunk: 'single',
+  // },
```

Comment:
- [ ] As mentioned in the best practices Do not commit old pieces of code as inline comments. They will make your project look messy. please remove all commented code from the codebase